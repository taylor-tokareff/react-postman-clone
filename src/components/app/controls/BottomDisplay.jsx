import React from 'react';
import styles from './BottomDisplay.css';
import PropTypes from 'prop-types';


const BottomDisplay = ({ body }) => {
  return <pre className={styles.back}>{JSON.stringify(body, null, 2)}</pre>;

};

BottomDisplay.propTypes = {
  body: PropTypes.string.isRequired
};

export default BottomDisplay;
