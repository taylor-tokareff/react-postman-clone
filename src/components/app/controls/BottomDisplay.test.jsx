import React from 'react';
import { render } from '@testing-library/react';
import BottomDisplay from './BottomDisplay.jsx';

describe('BottomDisplay component', () => {
  it('renders bottomDisplay', () => {
    const { asFragment } = render(
      <BottomDisplay
        body="name: none"
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
