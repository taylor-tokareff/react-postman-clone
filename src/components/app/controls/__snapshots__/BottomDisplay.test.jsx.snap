// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomDisplay component renders bottomDisplay 1`] = `
<DocumentFragment>
  <pre
    class="back"
  >
    "name: none"
  </pre>
</DocumentFragment>
`;
